src/Common/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -6,7 +6,7 @@ namespace System.Numerics.Hashing
 {
     internal static class HashHelpers
     {
-        public static readonly int RandomSeed = Guid.NewGuid().GetHashCode();
+        public static readonly int s_randomSeed = Guid.NewGuid().GetHashCode();
```

Comment:
We don't use the s_ naming for public fields.

Reply:
I didn't know that. We should add that as an exception to our [coding style guidelines]( https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style).